.gitignore

```diff
@@ -21,3 +21,4 @@ cover
 *.png
 
 .eggs
+docker/
```

Comment:
Maybe add a comment (for future reference) explaining why we are ignoring this folder ?